security.md

```diff
@@ -87,6 +88,16 @@ All security advisories made for Foreman are listed below with their correspondi
 
 ### Disclosure details
 
+#### <a id="2022-3874"></a>CVE-2022-3874: OS command injection via ct_command and fcct_command
+
+`ct_command` and `fcct_command` settings, available via Administer - Settings, both accept arbitrary
+strings as the command name and calling CoreOS templates will execute those commands as the user Foreman runs under.
+By default, only Foreman super administrator can access settings.
+
+* Affects Foreman 3.2.0 and higher
+* Fix released in Foreman 3.9.0
```

Comment:
Same question for you! 3.8? 3.7?

Reply:
3.8 probably yes, given it's not released yet.

older? imho not.